# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/dns/nsHostResolver.h
# Commit: 26c5be98ea21
# Full Hash: 26c5be98ea21c8c331cefe9649e3eb53197cd821
# Author: Kershaw Chang <kershaw@mozilla.com>
# Date: 2020-08-26 03:24:37
# Regressor Bug: 1652667
# File Overlap Count: 1
# Description:
#   Bug 1652667 - P2: Telemetry for whether a connection is succeeded by using an HTTPSSVC record r=valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D87857
# ==============================================================================

diff -r 96bd98319c62 -r 26c5be98ea21 netwerk/dns/nsHostResolver.h
--- a/netwerk/dns/nsHostResolver.h	Tue Aug 25 17:29:32 2020 +0000
+++ b/netwerk/dns/nsHostResolver.h	Tue Aug 25 17:29:31 2020 +0000
@@ -363,6 +363,7 @@
 
   // When the lookups of this record started (for telemetry).
   mozilla::TimeStamp mStart;
+  bool mAllRecordsExcluded = false;
 };
 
 NS_DEFINE_STATIC_IID_ACCESSOR(TypeHostRecord, TYPEHOSTRECORD_IID)